### ISSUE TITLE
The classes in the dependencies aar seems not to be converted

### ISSUE THREAD

**Issue by xulianbin** [evant/gradle-retrolambda#283]
My project has some of the dependencies which also contains the lambda statement.
When I applied this plugin, it was found the lambda code in the project is converted , but the code in the library seems not.
Is there any configuration to set so the plugin can change the byte code of the aar/jar files?

**Comment by evant** [comment]
Correct, it only applies to your project, not it's dependencies. You could try using Android's built-in desugaring which does.
